charts/templates/cluster-proxy-admin-rbac.yaml

```diff
@@ -0,0 +1,29 @@
+
+{{- if eq .Values.installClusterProxyRBAC "true" }}
+{{- $name := include "karmada.name" . -}}
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ $name }}-cluster-proxy-admin
+rules:
+  - apiGroups:
+      - 'cluster.karmada.io'
+    resources:
+      - clusters/proxy
+    verbs:
+      - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ $name }}-cluster-proxy-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ $name }}-cluster-proxy-admin
+subjects:
+  - kind: User
+    name: "system:admin"
+
+{{- end }}
```

Comment:
Add a blank line below the line.

Reply:
@XiShanYongYe-Chang  Why need a blank line below the line?

Reply:
I guess it is a encoding specification.

Reply:
@XiShanYongYe-Chang  I have modified it.

Reply:
> I guess it is a encoding specification.

It's like this.